requirements.txt

```diff
@@ -1,5 +1,5 @@
 anytree ~= 2.8
 colorama ~= 0.4
-oelint-parser ~= 2.9
+oelint-parser ~= 2.9.4
```

Comment:
Here I would suggest to use `oelint-parser ~= 2.9,oelint-parser >= 2.9.4`

Reply:
I changed my code like this. Thanks.